lib/CustomFields/pages/EditCustomFieldsRecord/EditCustomFieldsRecord.js

```diff
@@ -57,8 +77,14 @@ const EditCustomFieldsRecord = ({
   backendModuleId,
   backendModuleName,
   entityType,
-  fieldComponent,
   columnCount,
+  dispatch,
```

Comment:
Where does this component get access to ‘dispatch’ from?
I think it’ll be better if we’re not giving the component access to ‘dispatch’ directly but bind actions in ‘mapDispatchToProps’

Reply:
@BogdanDenis , makes sense!